spirv_cross.cpp

```diff
@@ -1468,6 +1468,39 @@ bool Compiler::get_binary_offset_for_decoration(VariableID id, spv::Decoration d
 	return true;
 }
 
+bool Compiler::block_is_noop(const SPIRBlock &block) const
+{
+	if (block.terminator != SPIRBlock::Direct)
```

Comment:
Should I include `SPIRBlock::Unreachable`? It's not necessary for `Compiler::block_is_loop_candidate` but one could argue an unreachable block should be no-op.